projects/transformers/run_utils.py

```diff
@@ -1128,5 +1264,49 @@ def model_init():
                 writer.write(f"{key} = {value}\n")
 
 
-def compute_objective_eval_loss(metrics):
-    return metrics["eval_loss"]
+def compute_objective(metrics, objective):
+    return metrics[objective]
+
+
+def check_if_current_hp_best(old_file, model_args, best_run):
```

Comment:
Might be good to add a docstring here or some more comments inside, it is a hard to read function